vcap.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+type Env struct {
+	SystemEnvJSON SystemEnv `json:"system_env_json"`
+}
+
+type SystemEnv struct {
+	VcapServices map[string][]Service `json:"VCAP_SERVICES"`
+}
+
+type Service map[string]interface{}
```

Comment:
You could make the parsing easier by using a struct here instead of `interface{}` eg:

```go
type Service struct {
    Credentials Credentials `json:"credentials"`
}
```

At the moment, it doesn't look as though the Credentials type is being used at all.

Reply:
I am unsure why this was changed!

I believe the idea was that it would be nice if this plugin could be a bit relaxed about exactly what comes back in the credentials (to avoid unmarshal errors and be more flexible for future services). But clearly that has not been explored and is not needed at this point in time.

So 🙌  :)